### PR TITLE
Aubert (FR, CH) (shop/baby_store) (Evermaps storefinder/sitemap+structured data) (141 stores)

### LINKED ISSUE #7039: Aubert (FR) (shop/baby_goods) (Evermaps Storefinder)
### Brand name

Aubert

French baby goods retail chain

### Wikidata ID

Q62964657
https://www.wikidata.org/wiki/Q62964657
https://www.wikidata.org/wiki/Special:EntityData/Q62964657.json

### Store finder url(s)

https://magasins.aubert.com/fr/france-FR

Official Url(s): https://www.aubert.com/
- [x] I have run the below and if anything in generated, added it.
- [x] I have manually checked for a storefinder
- [x] This is not a manufacturer listing outlets operating under their own brands / this will not create duplicates
`pipenv run scrapy sf --brand-wikidata=Q62964657 https://www.aubert.com/`
```
```

### PR DESCRIPTION
**Pull request by CloCkWeRX**
Fixes #7039

- [x] Add default tags?

Data appears sane:
```
2024-02-10 06:41:36 [scrapy.core.scraper] DEBUG: Scraped from <200 https://magasins.aubert.com/fr/suisse-CH/MT2/aubert-baby-2000-sion/details>
{'brand': 'Aubert',
 'brand_wikidata': 'Q62964657',
 'city': 'Sion',
 'country': 'CH',
 'email': None,
 'extras': {'@spider': 'aubert_fr', 'shop': 'baby_goods'},
 'facebook': 'https://www.facebook.com/Aubertbaby2000Sion/',
 'geometry': {'coordinates': [7.38388, 46.22737], 'type': 'Point'},
 'image': 'https://magasins.aubert.com/api/v1/medias/poi/one?mediaid=0ed5a72f-af81-431b-a16c-c406a5b108c9',
 'name': 'Aubert - BABY 2000 Sion (VS)',
 'nsi_id': 'aubert-e7ec7f',
 'opening_hours': 'Mo 14:00-18:30; Tu-Fr 09:00-12:00,14:00-18:30; Sa '
                  '09:00-17:00',
 'phone': '+41 27 323 81 80',
 'postcode': '1950',
 'ref': 'https://magasins.aubert.com/#location-MT2',
 'state': None,
 'street_address': 'Route du Manège, 60 Centre Commercial Art de Vivre À côté '
                   "d'Otto's",
 'website': 'https://magasins.aubert.com/fr/suisse-CH/MT2/aubert-baby-2000-sion/details'}
 ```
```
2024-02-10 06:52:21 [scrapy.statscollectors] INFO: Dumping Scrapy stats:
{'atp/brand/Aubert': 141,
 'atp/brand_wikidata/Q62964657': 141,
 'atp/category/shop/baby_goods': 141,
 'atp/field/email/missing': 141,
 'atp/field/operator/missing': 141,
 'atp/field/operator_wikidata/missing': 141,
 'atp/field/state/missing': 141,
 'atp/field/twitter/missing': 141,
 'atp/nsi/perfect_match': 141,
 'downloader/request_bytes': 432521,
 'downloader/request_count': 1079,
 'downloader/request_method_count/GET': 1079,
 'downloader/response_bytes': 16518509,
 'downloader/response_count': 1079,
 'downloader/response_status_count/200': 1079,
 'dupefilter/filtered': 4503,
 'elapsed_time_seconds': 660.47157,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2024, 2, 10, 6, 52, 21, 123042, tzinfo=datetime.timezone.utc),
 'httpcompression/response_bytes': 100047324,
 'httpcompression/response_count': 1079,
 'item_scraped_count': 141,
 'log_count/DEBUG': 1232,
 'log_count/INFO': 20,
 'memusage/max': 274911232,
 'memusage/startup': 147296256,
 'request_depth_max': 3,
 'response_received_count': 1079,
 'robotstxt/request_count': 1,
 'robotstxt/response_count': 1,
 'robotstxt/response_status_count/200': 1,
 'scheduler/dequeued': 1078,
 'scheduler/dequeued/memory': 1078,
 'scheduler/enqueued': 1078,
 'scheduler/enqueued/memory': 1078,
 'start_time': datetime.datetime(2024, 2, 10, 6, 41, 20, 651472, tzinfo=datetime.timezone.utc)}
2024-02-10 06:52:21 [scrapy.core.engine] INFO: Spider closed (finished)
```

### CODE REVIEW

File: locations/spiders/aubert_fr.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class AubertFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "aubert_fr"
+    item_attributes = {"brand": "Aubert", "brand_wikidata": "Q62964657", "extras": Categories.SHOP_BABY_GOODS.value}
+    download_delay = 0.5
```
Comment (iandees) [inline locations/spiders/aubert_fr.py:11]: Please don't set `download_delay` in the spider unless we need to make it much slower.

Comment (CloCkWeRX) [reply locations/spiders/aubert_fr.py:11]: No probs, was a copy/paste inclusion from another spider

Comment (CloCkWeRX) [inline locations/spiders/aubert_fr.py:11]:
```suggestion
```

**Comment by CloCkWeRX** [comment]
@iandees anything further needed for this one?

**Pull request merged**